doc/reference/power_management/index.rst

```diff
@@ -184,26 +172,26 @@ The power management subsystem classifies power management schemes
 into two categories based on whether the CPU loses execution context during the
 power state transition.
 
-* SYS_PM_LOW_POWER_STATE
-* SYS_PM_DEEP_SLEEP
+* Low Power State
+* Deep Sleep State
 
-SYS_PM_LOW_POWER_STATE
-======================
+Low Power State
+===============
 
 CPU does not lose execution context. Devices also do not lose power while
 entering power states in this category. The wake latencies of power states
 in this category are relatively low.
 
-SYS_PM_DEEP_SLEEP
-=================
+Deep Sleep State
+================
 
 CPU is power gated and loses execution context. Execution will resume at
 OS startup code or at a resume point determined by a bootloader that supports
 deep sleep resume. Depending on the SOC's implementation of the power saving
 feature, it may turn off power to most devices. RAM may be retained by some
 implementations, while others may remove power from RAM saving considerable
-power. Power states in this category save more power than
-`SYS_PM_LOW_POWER_STATE`_ and would have higher wake latencies.
+power. Power states in this category save more power than Low Power states
+and would have higher wake latencies.
```

Comment:
wake up?

Reply:
In whole document there is "wake latency" repeated multiple times. Should I change it here or everywhere?

Reply:
I would change this everywhere - "wake latency" does not make too much sense to me. I spotted it here, because this text was changed :) "wake-up latency" is the right term, IMO